definitions/infrastructure_as_a_service.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Infrastructure as a Service (IaaS)
+status: Completed
+category: Concept
+---
+
+## Infrastructure as a Service
+
+### What it is
+
+Infrastructure as a service i.e. IaaS is a [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) service model that offers [virtualized](https://github.com/cncf/glossary/blob/main/definitions/virtualization.md) compute, storage and network resources on-demand over the internet to consumers on a pay-as-you-go cost model.
+In IaaS model, cloud providers owns and operates the hardware and software, it is available in Public cloud, Private cloud and Hybrid cloud deployment options. 
+
+Comparison table
```

Comment:
I like this but it doesn't fit with the glossary style. Please remove the table.

Reply:
done